.travis.yml

```diff
@@ -1,12 +1,24 @@
 language: objective-c 
-osx_image: xcode11
+osx_image: xcode11.2
```

Comment:
isn't that still beta?

Reply:
oh true, thanks for the hint! I got tricked by the travis documentation where it is the latest xcode version: https://docs.travis-ci.com/user/reference/osx/#xcode-version